cli/lsp/documents.rs

```diff
@@ -633,6 +634,23 @@ struct FileSystemDocuments {
 }
 
 impl FileSystemDocuments {
+  pub fn get<'a>(
+    &mut self,
+    cache: &HttpCache,
+    get_maybe_resolver: impl Fn() -> Option<&'a dyn deno_graph::source::Resolver>,
```

Comment:
Does it actually need to be a closure? It seems it's used only in this function so should be okay to pass actual resolver?

Reply:
I looked into it and it seems ok to just create it each time. It's essentially just providing a reference. I'll change this.